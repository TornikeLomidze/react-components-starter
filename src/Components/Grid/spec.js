import React from 'react'
import Container from '.';
import { render } from '../../test.setup';
import renderer from 'react-test-renderer';


describe('Containers', () => {
  it('renders component without crashing', () => {
    const tree = renderer.create(<Container >Teste</Container>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});