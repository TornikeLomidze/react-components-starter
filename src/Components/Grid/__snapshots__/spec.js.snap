// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Rendering Container without crashing 1`] = `
<div>
  Text 2
</div>
`;
